packages/apps/helpdesk/src/screens/MyTeamTicketListScreen.js

```diff
@@ -49,16 +49,10 @@ const MyTeamTicketListScreen = ({navigation}) => {
   const [selectedStatus, setSelectedStatus] = useState([]);
   const [priorityStatus, setPriorityStatus] = useState([
     {
-      color: Ticket.getPriorityColor(Ticket.priority.Low, Colors),
-      isActive: true,
-      key: 1,
-      title: I18n.t('Helpdesk_Priority_Low'),
+      key: 3,
     },
     {
-      color: Ticket.getPriorityColor(Ticket.priority.Low, Colors),
-      isActive: true,
-      key: 2,
-      title: I18n.t('Helpdesk_Priority_Normal'),
+      key: 4,
     },
   ]);
```

Comment:
In the perspective to make it look cleaner, can you initialize the useState by doing a filter on active chips in the list returned from the type file please?